src/main/java/org/jabref/gui/maintable/RightClickMenu.java

```diff
@@ -74,6 +74,14 @@ public static ContextMenu create(BibEntryTableViewModel entry,
                 SpecialFieldMenuItemFactory.getSpecialFieldSingleItem(SpecialField.PRINTED, factory, () -> libraryTab, dialogService, preferencesService, undoManager, stateManager),
                 SpecialFieldMenuItemFactory.createSpecialFieldMenu(SpecialField.PRIORITY, factory, () -> libraryTab, dialogService, preferencesService, undoManager, stateManager),
                 SpecialFieldMenuItemFactory.createSpecialFieldMenu(SpecialField.READ_STATUS, factory, () -> libraryTab, dialogService, preferencesService, undoManager, stateManager),
+                factory.createMenuItem(
+                        // The new group is placed inside the currently active group, so
+                        // the button's name changes based on whether we are in the root group
+                        // or a subgroup.
```

Comment:
Althrough I like this, it is consistent to    The "All entries" group always has "Add subgroup". See screenshot https://github.com/JabRef/jabref/pull/11476#pullrequestreview-2170254425

One would have to change the whole menu to be consistent. This is too much effort. Thus, alwys add subgroup...